packages/terra-action-footer/tests/wdio/action-footer-spec.js

```diff
@@ -1,30 +1,30 @@
 Terra.describeViewports('ActionFooter', ['small', 'medium'], () => {
   it('displays ActionFooter with multiple end actions', () => {
-    browser.url('/#/raw/tests/terra-action-footer/action-footer/multiple-end-action-footer');
+    browser.url('/raw/tests/terra-core-docs/action-footer/multiple-end-action-footer');
```

Comment:
Since tests are shipped with the component, does this mean terra-core-docs is expected to be installed for testing?

Reply:
you'd need to yes, but i think we should discontinue the practice of shipping tests with components.

Reply:
I had considered moving the wdio tests to the docs package as well, but I ended up not wanting to mess with how we currently can run just the tests for a specific package.